colour/colorimetry/spectrum.py

```diff
@@ -3690,9 +3939,10 @@ def interpolate(self, shape=SpectralShape(), method=None):
 
     def align(self,
               shape,
-              method='Constant',
-              left=None,
-              right=None):
+              interpolation_method=None,
+              extrapolation_method='Constant',
+              extrapolation_left=None,
```

Comment:
It takes precedence on `Linear` or `Constant` methods: https://github.com/colour-science/colour/blob/6d112d1d3c969aa98c7faf071bf1795ad455ad73/colour/algebra/extrapolation.py#L298